src/Catalog/Helpers/GalleryDatabaseQueryService.cs

```diff
@@ -48,6 +51,44 @@ public class GalleryDatabaseQueryService : IGalleryDatabaseQueryService
                 Db2CatalogCursor.ByLastEdited(since, top));
         }
 
+        public async Task<FeedPackageDetails> GetPackage(string id, string version)
```

Comment:
>GetPackage [](start = 46, length = 10)

`GetPackageDetailsAsync` records a duration metric for each query it executes, consider adding that to `GetPackage` as well

Reply:
Added in https://github.com/NuGet/NuGet.Services.Metadata/pull/537/commits/ff3e15d46eda897db0d1ab14ce4a608cc31794c0